airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -25,6 +26,8 @@
 from airflow.providers.amazon.aws.hooks.redshift_data import RedshiftDataHook
 from airflow.providers.amazon.aws.utils import trim_none_values
 
+from mypy_boto3_redshift_data.type_defs import GetStatementResultResponseTypeDef
+
```

Comment:
Please move it under the `if TYPE_CHECKING:` block

Reply:
Fixed